### PR TITLE
Hi🅱️🅱️opotamous

### PR DESCRIPTION
**Pull request by oleglazari**
Setup for new project!

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Top level CMakeList for building the *EVT_BOARD_NAME* source code
+# Top level CMakeList for building the *EVT_HIB* source code
```
Comment (mjmagee991) [inline CMakeLists.txt]:
```suggestion
# Top level CMakeList for building the HIB source code
```

File: README.md [hunk 2]
```diff
@@ -1,78 +1,5 @@
...
-should be created to merge into main.
-
-# BIKE_NAME-BOARD_NAME
+# REV3-HIB
 
 ## Introduction
 
-*One-paragraph summary of the board and its purpose on the bike*
\ No newline at end of file
+Handlebar Interface Board (HIB). This board is capable of taking an analog input from the potentiometer based throttle and converting it to a digital signal with the use of an SPC58 microcontroller. This board is used to turn the throttle signal into a signal interpretable by the motor controller. There are ADCs for Throttle functionality, Braking functionality, and a button to start the motorcycle. All of these are feed into a Can loopback for validation.
```
Comment (mjmagee991) [inline README.md]:
```suggestion
Handlebar Interface Board (HIB). This board is capable of taking an analog input from the potentiometer based throttle and converting it to a digital signal with the use of an STM32 microcontroller. This board is used to read the analog throttle signal and transmit it as a CAN message. There are ADCs for throttle and brake functionality. The board also has CAN loopback hardware for self-testing.
```

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
A couple quick changes

**Review by mjmagee991 — APPROVED** [review]
Nice work!

**Review by ActuallyTaylor — APPROVED** [review]

**Pull request merged**